android/Vagrantfile

```diff
@@ -137,11 +137,11 @@ Vagrant.configure("2") do |config|
   end
 
   config.vm.provision "shell" do |s|
-      s.name = "Clone BOINC repository and configuring toolchain..."
+      s.name = "Cloning BOINC repository and configuring toolchain..."
       s.privileged  = false
       s.inline  = <<-SHELL
-          export OPENSSL_VERSION=1.0.2p
-          export CURL_VERSION=7.61.0
+          export OPENSSL_VERSION=1.0.2q
```

Comment:
You should update android/buildAndroidBOINC-CI.sh with both OPENSSL_VERSION and CURL_VERSION you use here

Reply:
That's your file 😈

No, honestly, I wasn't aware of it and strictly speaking it's independent of vagrant. Ideally that would be just one file but I'm not sure how to do that... But ok, I'll quickly update those.

Reply:
Done

Reply:
Yes, it's mine but they are doind almost the same so it is nice to keep them both updated.
I have some thoughts how to keep these versions in one place but I need a little bit more time to convert these thoughts into code :smile: